server/controllers/finance/cash.js

```diff
@@ -140,19 +137,20 @@ function detail(req, res, next) {
 
 /**
  * POST /cash
- * Creates a cash payment against several sales or a cautionary payment.  The
- * API also supports future offline functionality by either accepting a UUID or
- * generating it if it is not present.
+ * Creates a cash payment against one or many previous sales or a cautionary
+ * payment.  If a UUID is not provided, one is automatically generated.
  */
 function create(req, res, next) {
   'use strict';
 
-  var data = req.body.payment;
+  // alias insertion data
+  const data = req.body.payment;
```

Comment:
Your use of ```const``` is so abusive, even if a constant is a value which will not be changed during the execution, but logic needs that a constant must be known and must remain the same in every situation.
so for example the area of circle is ```area = 3.14 * (R*R)``` in this case ```3.14``` is constant because it is known and will not change no mater the situation but ```R``` is the variable, because it depend on the problem, it can be ```1, 2, 1.6, or 30 ...```.